src/structures/Guild.js

```diff
@@ -573,7 +573,9 @@ class Guild {
    * be resolved, the user ID will be the result.
    * @example
    * // ban a user
-   * guild.ban('123123123123');
+   * guild.ban(user)
```

Comment:
I agree, expansion on what `user` is could be beneficial, but this is still an improvement on `123123123`.
As for bans being invoked before the user is in the guild, that's news to me. Do you know of a way to resolve a user outside of any guilds the bot is in?